cloudformation/ebs-substack.cfn.json

```diff
@@ -121,6 +121,19 @@
     },
     "Vol1_UseEBSPIOPS": {
       "Fn::Or": [
+        {
```

Comment:
nit: Did you modify this template manually? Ideally you should be modifying `util/cfn-stacks-generators/generate-ebs-substack.py` and using that to generate the template. Same with the RAID substack

We have those generator because it is difficult to maintain these repetitive substacks by hand. Let's try to use them if we can. Is there an issue that prevents us from using those generators?

Reply:
Maybe the generator is out of date